plugins/Goals/lang/en.json

```
@@ -68,30 +68,30 @@
+        "GoalConversionsBy": "%s goal conversions by type of visit",
+        "GoalIsTriggered": "A goal is triggered",
+        "GoalIsTriggeredWhen": "A goal is triggered when",
         "GoalName": "Goal Name",
         "Goals": "Goals",
         "NGoals": "%s goals",
         "NRevenue": "%s revenue",
         "NItems": "%s items",
         "ManageGoals": "Manage Goals",
-        "GoalsOverview": "Goals Overview",
-        "GoalsOverviewDocumentation": "This is an overview of your goal conversions. Initially, the graph shows the sum of all conversions. %s Below the graph, you can see conversion reports for each of your goals. The sparklines can be enlarged by clicking on them.",
-        "GoalX": "Goal %s",
-        "HelpOneConversionPerVisit": "If a Page matching this Goal is refreshed or viewed more than once in a Visit, the Goal will only be tracked the first time the page was loaded during this visit.",
+        "GoalsOverview": "Overview of Goals",
+        "GoalsOverviewDocumentation": "An overview of your goal conversions. Initially, the graph shows the sum of all conversions. %s Below the graph, you can see conversion reports for each of your goals. The sparklines can be enlarged by clicking on them.",
+        "GoalX": "%s Goal",
+        "HelpOneConversionPerVisit": "If a Page matching this goal is refreshed or viewed more than once during a visit, the goal will only be tracked the first time the page was loaded during the visit.",
         "IsExactly": "is exactly %s",
-        "LearnMoreAboutGoalTrackingDocumentation": "Learn more about %1$s Tracking Goals in Matomo%2$s in the user documentation.",
-        "LeftInCart": "%s left in cart",
+        "LearnMoreAboutGoalTrackingDocumentation": "Learn more about %1$stracking goals%2$s in Matomo in the user documentation.",
+        "LeftInCart": "%s left in the cart",
         "Manually": "manually",
-        "ManuallyTriggeredUsingJavascriptFunction": "Goal is manually triggered using the JavaScript API trackGoal()",
-        "MatchesExpression": "matches the expression %s",
-        "NewGoalIntro": "Goal Conversion tracking is one of the most efficient ways to measure and improve your business objectives.",
-        "NewVisitorsConversionRateIs": "New visitors conversion rate is %s",
-        "NoGoalsNeedAccess2": "Only a Write user, an Administrator or a user with Super User access can manage Goals for a given website. Please ask your Matomo administrator to set up a Goal for your website. <br>Tracking Goals is a great way to help understand and maximize your website performance!",
+        "ManuallyTriggeredUsingJavascriptFunction": "A goal is manually triggered using the JavaScript API trackGoal()",
+        "MatchesExpression": "matches the %s expression",
+        "NewGoalIntro": "Tracking of goal conversions is one of the most efficient ways to measure and improve your business objectives.",
+        "NewVisitorsConversionRateIs": "The new visitor conversion rate is %s",
```

This can now be interpreted wrong. it's not the `new *visitor conversion rate*`, but the `conversion rate of new visitors`
```suggestion
        "NewVisitorsConversionRateIs": "The conversion rate for new visitors is %s",
```